DynatraceAppMon/Context.cs

```diff
@@ -250,6 +254,89 @@ internal bool goToMethod(Lookup lookup)
             }
         }
 
+        private bool HasClassFiles(ProjectItem projectItem)
+        {
+            bool classFileOccurred = false;
+            for (int i = 1; i <= projectItem.FileCount; i++)
+            {
+                string fileExtension = Path.GetExtension(projectItem.FileNames[(short)i]);
+                if (fileExtension != null && (fileExtension.Contains(".cs") || fileExtension.Contains(".vb")))
+                {
+                    classFileOccurred = true;
```

Comment:
you may return true here, false on the end of method instead using variable and break